### PR TITLE
add ember-data with firebase to getting started

### PR DESCRIPTION
**Pull request by kellywerks**
Part of #445

### CODE REVIEW

File: data/pages.yml [hunk 1]
```diff
@@ -10,8 +10,8 @@
   pages:
     - title: "Installing Ember"
       url: "index"
-    - title: "Your First Ember App"
-      url: "first-app"
```
Comment (michaelrkn) [inline data/pages.yml]: Why is this deleted in this commit? I think this was deleted in another PR, too.

**Comment by locks** [comment]
Can you rebase?
